docs/maps/vector-style.asciidoc

```diff
@@ -10,12 +12,57 @@
 
 *Symbol size*:: Defines the symbol size of point features.
 
-Click the *link* button to toggle between static styling and data-driven styling.
+NOTE: *LineString* and *MultiLineString* geometries do not have fill and do not use fill color property.
+Set border color to style line geometries
+
+[float]
+[[maps-vector-style-static]]
+==== Static styling
+
+Use static styling to specificy a constant value for a style property.
 
 [role="screenshot"]
 image::maps/images/vector_style_static.png[]
 
+[float]
+[[maps-vector-style-data-driven]]
+==== Data driven styling
```

Comment:
### Suggestions

- Shorten section about sample data
- Use bulleted list to explain properties
- 

<img width="793" alt="Screen Shot 2019-04-15 at 1 14 55 PM" src="https://user-images.githubusercontent.com/33642766/56162522-176dc300-5f81-11e9-9084-e39e721eb100.png">